help.go

```diff
@@ -46,11 +46,22 @@ func renderFlagHelp(fullName string, fl *pflag.FlagSet, w io.Writer) {
 
 // renderHelp generates a command's help page.
 func renderHelp(fullName string, cmd Command, fl *pflag.FlagSet, w io.Writer) {
+	spec := cmd.Spec()
+	usage := fmt.Sprintf("Usage: %v %v\n\n", fullName, spec.Usage)
+
+	// If the command has aliases, add them to the output as a comma-separated list.
+	if spec.HasAliases() {
+		usage += "Aliases: "
```

Comment:
Could we use a https://golang.org/pkg/strings/#Builder here instead of adding to the string in a loop?